G4EMPH/ParticleListAction.cxx

```diff
@@ -70,6 +71,8 @@ namespace emph {
   {
     fEnergyCut              = pset.get< double >("G4EnergyThreshold")*CLHEP::GeV;
     fManyParticles          = pset.get< bool   >("ManyParticles");
+
+    fMinTrajPtDist          = pset.get< double >("MinTrajPtDist",0.); 
```

Comment:
does this need the default value here when it is already set on line 58 (and the fhicl should have a value